microceph/api/disks.go

```diff
@@ -56,7 +58,16 @@ func cmdDisksPost(s *state.State, r *http.Request) response.Response {
 
 	mu.Lock()
 	defer mu.Unlock()
-	err = ceph.AddOSD(s, req.Path, req.Wipe, req.Encrypt)
+
+	data := types.DiskParameter{req.Path, req.Encrypt, req.Wipe}
```

Comment:
NIT: "data" definition should be consistent with wal/db object initialisation.

Reply:
Data can never be nil, hence why it's not a pointer.